docs/apidoc/watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.rst

```diff
@@ -0,0 +1,75 @@
+--------------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.SepRO_plus_Chlorination
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.ZeroDRO\_plus\_Chlorination module
+----------------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.ZeroDRO_plus_Chlorination
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.posttreatment\_ideal\_naocl\_chlorination\_block module
+-------------------------------------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.posttreatment_ideal_naocl_chlorination_block
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.pretreatment\_stoich\_softening\_block module
+---------------------------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.pretreatment_stoich_softening_block
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.test\_chlorination\_example module
+----------------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.test_chlorination_example
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.test\_posttreatment module
+--------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.test_posttreatment
+   :members:
+   :show-inheritance:
+
+watertap.examples.flowsheets.full\_treatment\_train.flowsheet\_components.chemistry.test\_pretreatment module
+-------------------------------------------------------------------------------------------------------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry.test_pretreatment
+   :members:
+   :show-inheritance:
+
+Module contents
+---------------
+
+.. automodule:: watertap.examples.flowsheets.full_treatment_train.flowsheet_components.chemistry
+   :members:
+   :show-inheritance:
```

Comment:
I don't think these should be made into documentation, unless we want them to be part of the proper set of tutorials. 

Some of these files include older test flowsheets that were not even part of the milestone.